curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62cc5b1779e4d313466f73c5.md

```diff
@@ -0,0 +1,72 @@
+---
+id: 62cc5b1779e4d313466f73c5
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+Nest a self-closing `link` element within the `head` element. Give it a `rel` attribute with value of `stylesheet` and an `href` attribute with a value of `styles.css`.
+
+# --hints--
+
+Your code should have a `link` element.
+
+```js
+assert.exists(document.querySelector('link'));
+```
+
+You should not change your existing `head` tags. Make sure you did not delete the closing tag.
+
+```js
+const heads = document.querySelectorAll('head');
+assert.equal(heads?.length, 1);
+```
+
+You should have one self-closing `link` element.
+
+```js
+assert(document.querySelectorAll('link').length === 1);
+```
+
+Your `link` element should be within your `head` element.
+
+```js
+assert.exists(document.querySelector('head > link'));
+```
+
+Your `link` element should have a `rel` attribute with the value `stylesheet`.
+
+```js
+const link_element = document.querySelector('link');
+const rel = link_element.getAttribute("rel");
+assert.equal(rel, "stylesheet");
+```
+
+Your `link` element should have an `href` attribute with the value `styles.css`.
+
+```js
+const link = document.querySelector('link');
+assert.equal(link.dataset.href, "styles.css");
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Registration Form</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+--fcc-editable-region--
+  <body>
+
+  </body>
+--fcc-editable-region--
```

Comment:
```suggestion
--fcc-editable-region--
    <head>
    <meta charset="UTF-8">
    <title>Registration Form</title>
  </head>
--fcc-editable-region--

    <body>
  </body>

```

The seed code here includes the solution for the new step and the editable regions aren't in the right spot - quite sure anyway. Maybe something like this ^

Reply:
I have edited your comment because `body` won't be changed. 
As usual thanks for catching these out

Reply:
I included the body in there because the step before this has it

Reply:
Changed step 4, as well. 👍